opa/rego/rules/untrusted_checkout_exec.rego

```diff
@@ -91,3 +91,32 @@ _workflows_runs_from_pr contains [pkg.purl, workflow] if {
 
 	utils.filter_workflow_events(parent, github.workflow_run.parent.events)
 }
+
+# Azure Devops
+
+results contains poutine.finding(rule, pkg.purl, {
+    "path": pipeline.path,
+    "job": job.job,
+    "step": step_id,
+    "line": step.lines[attr],
+	"details": sprintf("Detected usage of `%s`", [cmd]),
+}) if {
+	pkg := input.packages[_]
+    pipeline := pkg.azure_pipelines[_]
+    is_untrusted_checkout_azure(pipeline)
+    job := pipeline.stages[_].jobs[_]
+    step := job.steps[step_id]
```

Comment:
We are missing the sequential check. The command must be after the checkout. If it’s before then it probably would not use the content of the pr head